content/attacks/posts/2023-11-23-HecoBridge.md

```diff
@@ -47,19 +47,12 @@ HTX:
    - $2,195,836 in ARIX
    - $4,254,541 in KOK
```

Comment:
Couldnt understand the meaning of ARIX and KOK, if it's tokens, it's unnecessary to include them.

Reply:
Kok Play is a decentralized digital content platform and Arix is an Exchange. Should I add the details?